samples/estimation/estimation-frontier-widgets.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import qsharp\n",
+    "from qsharp_widgets import SpaceChart, EstimateDetails, Summary\n",
+    "with open(\"Dynamics.qs\", \"r\") as f:\n",
+    "    contents = f.read()\n",
+    "qsharp.eval(contents)\n",
+    "\n",
+    "# esitmate for the shortest runtime and the default qubit\n",
+    "result1 = qsharp.estimate(\"QuantumDynamics.Main()\")"
```

Comment:
Do you do anything with result1? I don't see it referenced anywhere else. It seems to be dead code/unused and this line should be removed.